src/app/screens/LNURLWithdraw/index.tsx

```diff
@@ -61,11 +61,18 @@ function LNURLWithdraw(props: Props) {
 
   function renderAmount() {
     if (minWithdrawable === maxWithdrawable) {
-      return <p>{`${minWithdrawable / 1000} sats`}</p>;
+      return (
+        <ContentMessage
+          heading={`Amount (Satoshi)`}
+          content={`${minWithdrawable / 1000} sats`}
+        />
+      );
     } else {
       return (
-        <div className="mt-1 flex flex-col">
-          <Input
+        <div className="my-4 p-4 shadow bg-white dark:bg-surface-02dp rounded-lg overflow-hidden">
+          <TextField
```

Comment:
should this even be a dual currency field? (that shows fiat amounts?)

Reply:
Good point. @im-adithya wanna give that a try?

Reply:
Sure, initially thought of using that, will change now!